wallet_ui/components/panels/CreateCanister.tsx

```diff
@@ -179,31 +175,31 @@ export function CreateCanisterDialog(props: {
             <div style={{display: "flex"}}>
               <TextField
                 label="Controller"
-                value={controller} //should default controller(walletId) be editable in the first place?
+                value={controllersState[0]} //should default controller(walletId) be editable in the first place?
                 style={{ margin: "8px 0 24px" }}
                 fullWidth
                 disabled={loading}
-                onChange={handleControllerChange}
-                error={error1}
+                onChange={(event) => handleInputChange(0, event)}
+                error={error[0]}
                 autoFocus
                 InputLabelProps={{ shrink: true }}
               />
               <Button onClick={increaseInput}>
                 <AddIcon/>
               </Button>
             </div>
-            {moreControllers.map((field : string, ind : number) => (
-              <div key={ind} style={{display: "flex", marginBottom: "10px"}}>
+            {controllersState.slice(1).map((field: string, ind: number) => (
+              <div key={ind + 1} style={{display: "flex", marginBottom: "10px"}}>
```

Comment:
I'm guessing you're using `ind + 1` so that React stops yelling at you. It's not really a big issue here, but it can lead to weird problems.

Also, I usually use `index` or `idx` when I want to abbreviate